### ISSUE TITLE
[TW-1580] modifying a task doesn't change modification time

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1606]
_David Patrick on 2015-03-23T10:07:04Z says:_ 

 Like the subject line says, mods to a task don;t necessarily change the modification time, as evidenced by this little test:

djp(at)tigger:~$ date
Mon Mar 23 04:38:17 EDT 2015
djp(at)tigger:~$ tw 133 mod -clean
Modifying task 133 'renovate manse'.
Modified 1 task.
Project 'manse' is 28% complete (5 of 7 tasks remaining).
djp(at)tigger:~$ tw 133 export
{"id":133,"description":"renovate manse","entry":"20150319T064039Z","modified":"20150319T064039Z","project":"manse","status":"pending","uuid":"fe765047-06b6-4205-8e16-cac06bcec420","annotations":[{"entry":"20150319T064207Z","description":"[ ] fix ceilings"},{"entry":"20150319T064312Z","description":"[ .."}],"urgency":7.92192}
djp(at)tigger:~$ date
Mon Mar 23 04:38:45 EDT 2015

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-03-23T10:07:04Z
Modified: 2015-03-23T15:40:53Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-03-23T15:40:53Z says:_ 

 Fixed in [commit 4f75652ccb3e32a5cd6e3bfaa75c52b40f1e33cf|https://git.tasktools.org/projects/TM/repos/task/commits/4f75652ccb3e32a5cd6e3bfaa75c52b40f1e33cf]. The problem was present in 2.4.0, 2.4.1 and 2.4.2.
